src/models/piranha_output.rs

```diff
@@ -16,21 +16,31 @@ use std::path::PathBuf;
 use itertools::Itertools;
 use serde_derive::Serialize;
 
+use crate::utilities::gen_py_str_methods;
+
 use super::{edit::Edit, matches::Match, source_code_unit::SourceCodeUnit};
-use pyo3::prelude::pyclass;
+use pyo3::{prelude::pyclass, pymethods};
+
+/// A class to represent Piranha's output
 #[derive(Serialize, Debug, Clone, Default)]
 #[pyclass]
 pub struct PiranhaOutputSummary {
+  /// path to the file
```

Comment:
```suggestion
  /// Path to the file
```

Is there any convention on whether these should be start with lower or uppercase? If not, I'd use "Path" unless you specifically mean the field/variable name.